packages/frontend/src/ee/features/aiCopilot/components/ChatElements/AgentChatBubbles.tsx

```diff
@@ -174,25 +220,41 @@ export const AssistantBubble: FC<{
                         </ActionIcon>
                     )}
                 </CopyButton>
-                {/* TODO: Add up/down vote to Web UI */}
-                <Group style={{ display: 'none' }}>
-                    <ActionIcon
-                        variant="subtle"
-                        color={upVoted ? 'green' : 'gray'}
-                        aria-label="upvote"
-                    >
-                        <IconThumbUp size={16} />
-                    </ActionIcon>
 
-                    <ActionIcon
-                        variant="subtle"
-                        color={downVoted ? 'red' : 'gray'}
-                        aria-label="downvote"
-                    >
-                        <IconThumbDown size={16} />
-                    </ActionIcon>
+                <Group gap={2}>
+                    {(!hasRating || upVoted) && (
+                        <ActionIcon
+                            variant={'transparent'}
+                            color="gray"
+                            aria-label="upvote"
+                            onClick={handleUpvote}
```

Comment:
purposely not adding a loading state bc I don't think it looks great with this action icon.